### PR TITLE
Trade endpoints

### PR DESCRIPTION
**Pull request by mcdayoub**

### CODE REVIEW

File: polygon/rest/models/trades.py [hunk 1]
```diff
@@ -21,3 +21,74 @@ class Trade:
...
+
+    @staticmethod
+    def from_dict(d):
+        return Last(
+            d.get("conditions", None),
+            d.get("exchange", None),
+            d.get("price", None),
+            d.get("size", None),
+            d.get("timestamp", None),
+        )
```
Comment (clickingbuttons) [inline polygon/rest/models/trades.py:77]: Can't you do `Last(**d)`?

File: polygon/rest/models/trades.py [hunk 2]
```diff
@@ -21,3 +21,74 @@ class Trade:
...
+    request_id: str
+
+    @staticmethod
+    def from_dict(d):
+        return LastTradeCrypto(
+            d.get("last", None),
+            d.get("symbol", None),
+            d.get("status", None),
+            d.get("request_id", None),
+        )
```
Comment (clickingbuttons) [inline polygon/rest/models/trades.py:94]: Can't you do `LastTradeCrypto(**d)`?

File: polygon/rest/base.py [hunk 3]
```diff
@@ -67,8 +67,9 @@ def _get(
 
         if result_key:
             obj = obj[result_key]
-        else:
-            # If the result_key does not exist, still need to put the results in a list
+
+        # If obj is not yet a list, need to turn it into a list
+        if type(obj) != list:
             obj = [obj]
```
Comment (clickingbuttons) [inline polygon/rest/base.py:73]: Why is this needed? I think it's okay if this method doesn't always return a list.

**Review by clickingbuttons — CHANGES_REQUESTED** [review]

File: polygon/rest/models/trades.py (hunk 2 above)
Comment (mcdayoub) [reply polygon/rest/models/trades.py:94]: symbol is different than ticker

File: polygon/rest/base.py (hunk 3 above)
Comment (mcdayoub) [reply polygon/rest/base.py:73]: last trade does not work without this

File: polygon/rest/models/trades.py (hunk 1 above)
Comment (mcdayoub) [reply polygon/rest/models/trades.py]: done

File: polygon/rest/base.py (hunk 3 above)
Comment (clickingbuttons) [reply polygon/rest/base.py:73]: Ah, I never expected "results" to not be a list. This is good.

**Review by clickingbuttons — APPROVED** [review]
